src/main/java/openfasttrack/cli/MissingArgumentException.java

```diff
@@ -0,0 +1,33 @@
+package openfasttrack.cli;
+
+/*
+ * #%L
+ * OpenFastTrack
+ * %%
+ * Copyright (C) 2016 hamstercommunity
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+public class MissingArgumentException extends CliException
```

Comment:
Using exceptions to control the program flow here is a border case. Wrong user input is not really an exception but a regular situation. Nevertheless here it makes handling the code actually cleaner - so: approved.